tensorflow/python/pywrap_tensorflow.py

```diff
@@ -46,8 +46,12 @@
     sys.setdlopenflags(_default_dlopen_flags)
 except ImportError:
   msg = """%s\n\nFailed to load the native TensorFlow runtime.\n
-See https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#import_error\n
-for some common reasons and solutions.  Include the entire stack trace
+Visit https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/install/index.md
```

Comment:
Can you update the URL to https://www.tensorflow.org/install/, which is prettier and pushed per release, so more stable?